bank-of-codepath-express-api/models/bank.js

```diff
@@ -1,84 +1,123 @@
-const { BadRequestError } = require("../utils/errors")
-const { storage } = require("../data/storage")
+const { BadRequestError } = require("../utils/errors");
+const { storage } = require("../data/storage");
 
 class Bank {
   static async listTransactions() {
     // list all items in the transactions array
-    const transactions = storage.get("transactions").value()
-    return transactions
+    const transactions = storage.get("transactions").value();
+    return transactions;
   }
 
   static async listTransfers() {
     // list all items in the transfers array
-    const transfers = storage.get("transfers").value()
-    return transfers
+    const transfers = storage.get("transfers").value();
+    return transfers;
   }
 
   static async fetchTransactionById(transactionId) {
     // fetch a single transaction
     const transaction = storage
       .get("transactions")
-      .find({ id: Number(transactionId) })
-      .value()
-    return transaction
+      .find({
+        id: isNaN(Number(transactionId))
```

Comment:
isNaN(Number(transferId)) ? transferId : Number(transferId)

this can be refactor into util function

const checkisNaN = (val) => isNaN(Number(val)) ? transferId : Number(val);